generators/pythonv2/codegen/src/ast/__test__/Class.test.ts

```diff
@@ -7,4 +7,14 @@ describe("class", () => {
         });
         expect(clazz.toString()).toMatchSnapshot();
     });
+
+    it("variables with annotation and initializer", async () => {
+        const clazz = python.class_({
+            name: "Car"
+        });
+        clazz.addVariable(
+            python.variable({ name: "color", type: python.annotation({ type: "str" }), initializer: "'red'" })
```

Comment:
[oos] interesting question of whether we should be outputting `'red'` or `"red"`. Maybe there's a prettier/black postprocessing step we could do using the user's config if there is one, defaulting to our own?

Reply:
Hm yeah interesting. My guess would be that we output the latter but then also have a postprocessing step, as commented above.

Reply:
(commented on another thread to use ruff fmt)